test/Exporter/Test/Writer/KnpSnappyWriterTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Source;
+
+use Exporter\Writer\KnpSnappyWriter;
+
+/**
+ * Tests the KnpSnapyWriter.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class KnpSnappyWriterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string
+     */
+    private $filename;
+
+    /**
+     * @var string
+     */
+    private $wkhtmltopdfBinary;
+
+    public function setUp()
+    {
+        $this->filename = 'foobar.pdf';
+        // $this->wkhtmltopdfBinary = __DIR__.'/../../../../vendor/h4cc/wkhtmltopdf-amd64/bin/wkhtmltopdf-amd64';
+        $this->wkhtmltopdfBinary = '/usr/local/bin/wkhtmltopdf';
```

Comment:
> Could not this be configurable?

Regarding the test only or the writer `wkhtmltopdfBinary` argument itself? 
For the test, I can set the binary path as an environment variable or something like to avoid the hard-coded path. If you have a better suggestion...